packages/runtime/src/interpreter/opcode.ts

```diff
@@ -193,13 +193,14 @@ export abstract class Op {
 	 * @param index Index
 	 * @param line line number in TEAL file
 	 */
-	assert64BitIndex(index: bigint, line: number): void {
+	assert64BitIndex(index: bigint, line: number): bigint {
```

Comment:
Why?? You are returning the same thing you passed to this method.

Reply:
Please think about this. You shouldn't return what you passed in method.

Reply:
this is how it is done in most asserts in op class